common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgBlueprintStore.java

```diff
@@ -15,7 +15,13 @@
  * #L%
  */
 
-import com.google.gson.*;
+//import com.google.gson.*;
```

Comment:
Delete instead of commenting here and down the imports

Reply:
ok..will remove